src/cli.py

```diff
@@ -0,0 +1,71 @@
+import sys
+import click
+from pathlib import Path
+from src.comparsion import compare_and_output
+
+def validate_paths(input_path: str, output_path: str, whichtwo) -> None:
+    input_path = Path(input_path)
+    codeQL_directory = Path(input_path, "codeQL_data")
+    clojure_directory = Path(input_path, "clojure_data/atomsClojure.csv")
```

Comment:
I assume that for Clojure input, there is still only one csv file? In that case, it could be an simple fix we just read from that csv file regardless of its name